### ISSUE TITLE
ERR_INVALID_CALLBACK()

### ISSUE THREAD

**Issue by ghost** [mmckegg/msi-packager#17]
On ubuntu 19.04 installed following packages: `msitools`, `wixl`, `nodejs`, `npm`  

Then installed msi-packager with: `sudo npm install -g msi-packager`

Executed the command:
`msi-packager ~/MYPROD/ ~/MYPROD.msi -n MYPROD -v 1.0.0 -m COMPANY -u 10000 -i ~/MYPROD/product.ico -e ~/MYPROD/client.exe`

Error message:
```
fs.js:129
  throw new ERR_INVALID_CALLBACK();
  ^

TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at maybeCallback (fs.js:129:9)
    at Object.write (fs.js:536:14)
    at /usr/local/lib/node_modules/msi-packager/index.js:33:10
    at /usr/local/lib/node_modules/msi-packager/generate-xml.js:15:5
    at /usr/local/lib/node_modules/msi-packager/generate-xml.js:166:7
    at /usr/local/lib/node_modules/msi-packager/node_modules/async-each/index.js:24:44
    at /usr/local/lib/node_modules/msi-packager/generate-xml.js:121:13
    at /usr/local/lib/node_modules/msi-packager/generate-xml.js:166:7
    at /usr/local/lib/node_modules/msi-packager/node_modules/async-each/index.js:24:44
    at /usr/local/lib/node_modules/msi-packager/generate-xml.js:121:13
```

**Comment by tayler0101** [comment]
did you found any solution?

**Comment by mathieu-aubin** [comment]
https://stackoverflow.com/questions/50438592/typeerror-err-invalid-callback-callback-must-be-a-function

**Comment by rmp** [comment]
should be resolved with latest v1.3.1 on github
